import-export-cli/cmd/deprecated/exportAPI.go

```diff
@@ -59,6 +61,9 @@ var ExportAPICmdDeprecated = &cobra.Command{
 	Deprecated: "instead use \"" + cmd.ExportCmdLiteral + " " + cmd.ExportAPICmdLiteral + "\".",
 	Run: func(deprecatedCmd *cobra.Command, args []string) {
 		utils.Logln(utils.LogPrefixInfo + exportAPICmdLiteral + " called")
+		if exportRevisionNum == "" {
+			fmt.Println("Revision number not provided. Only the working copy without deployment environments will be exported")
```

Comment:
Shall we change this as follows?
```suggestion
			fmt.Println("The revision number is not provided. Only the working copy without deployment environments will be exported.")
```

Reply:
Fixed with cf58769